Client/Assets/FindInPage.js

```diff
@@ -162,10 +165,32 @@ function updateHighlightedSpan() {
   var left = rect.left + scrollX - window.innerWidth / 2;
   left = clamp(left, 0, document.body.scrollWidth);
   top = clamp(top, 0, document.body.scrollHeight);
-  window.scrollTo(left, top);
+  scrollToSelection(left, top, SCROLL_INTERVAL_DURATION);
   debug("Scrolled to: " + left + ", " + top);
 }
 
+function scrollToSelection(left, top, duration) {
+  var time = 0,
+  startX = scrollX,
+  startY = scrollY;
+  clearInterval(scrollInterval);
+  scrollInterval = setInterval(function() {
+    if (time < duration) {
```

Comment:
You don't need this -- if you're still in the interval, that means you haven't cleared it yet, so you know time will be <= duration.